.github/workflows/node-ci.yml

```diff
@@ -32,167 +32,37 @@ jobs:
+          key: ${{ runner.os }}-node${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
-            ${{ runner.os }}-yarn-
+            ${{ runner.os }}-node${{ matrix.node-version }}-yarn-
 
       - uses: actions/cache@v2
         id: sewing-kit-cache
         name: Restore sewing-kit cache
         with:
-          path: |
-            .sewing-kit/
-          key: ${{ runner.os }}-sewing-kit-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-sewing-kit-
-
-      - name: 📦 Install dependencies
-        run: yarn --production=false --frozen-lockfile
-
-      - name: 💅🏼 Lint
-        run: |
-          yarn lint
-          yarn ci:lint-docs
-
-  node-build:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
-
-    steps:
-      - uses: actions/checkout@v2
-        name: Checkout
-
-      - uses: actions/setup-node@v1
-        name: Use Node.js ${{ matrix.node-version }}
-        with:
-          node-version: ${{ matrix.node-version }}
-
-      - name: Get yarn cache directory
-        id: yarn-cache-get-dir
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - uses: actions/cache@v2
-        id: yarn-cache
-        name: Restore yarn cache
-        with:
-          path: ${{ steps.yarn-cache-get-dir.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-
-      - uses: actions/cache@v2
-        id: build-cache
-        name: Restore build cache
-        with:
-          path: |
-            .sewing-kit/
-            ./packages/*/build/
-          key: ${{ runner.os }}-${{ matrix.node-version }}-build-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.node-version }}-build-
-
-      - name: 📦 Install dependencies
-        run: yarn --production=false --frozen-lockfile
-
-      - name: 🔨 Build
-        run: yarn build
-
-  node-e2e-tests:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
-
-    steps:
-      - uses: actions/checkout@v2
-        name: Checkout
-
-      - uses: actions/setup-node@v1
-        name: Use Node.js ${{ matrix.node-version }}
-        with:
-          node-version: ${{ matrix.node-version }}
-
-      - name: Get yarn cache directory
-        id: yarn-cache-get-dir
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - uses: actions/cache@v2
-        id: yarn-cache
-        name: Restore yarn cache
-        with:
-          path: ${{ steps.yarn-cache-get-dir.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-
-      - uses: actions/cache@v2
-        id: build-cache
-        name: Restore build cache
-        with:
-          # index packages files need to be cached for e2e tests
           path: |
```

Comment:
I don't really like this cache config. Restoring build output feels like it is an avenue to false confidence where we think content is being generated but it's actually just restored from the cache.